src/java.base/share/classes/sun/net/www/http/KeepAliveStreamCleaner.java

```diff
@@ -64,22 +62,10 @@ class KeepAliveStreamCleaner
 
     static {
         final String maxDataKey = "http.KeepAlive.remainingData";
-        @SuppressWarnings("removal")
-        int maxData = AccessController.doPrivileged(
-            new PrivilegedAction<Integer>() {
-                public Integer run() {
-                    return NetProperties.getInteger(maxDataKey, 512);
-                }}).intValue() * 1024;
-        MAX_DATA_REMAINING = maxData;
+        MAX_DATA_REMAINING = NetProperties.getInteger(maxDataKey, 512) * 1024;
```

Comment:
I'm don't know the context, but `Math.multiplyExact()` can be a safer alternative.

Reply:
What happens if we overflow is that we will not try to discard any data from the keep alive stream, exactly as if a negative or zero value had been provided. That said, the KeepAliveStream compares that with `expected`, which is a long, so `MAX_DATA_REMAINING` should really be a long, not an int, and given that the property value is an int, that would prevent any overflow. I might just do that.